src/ActiveLogin.Authentication.BankId.AspNetCore.Qr/ActiveLogin.Authentication.BankId.AspNetCore.Qr.csproj

```diff
@@ -0,0 +1,57 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <NeutralLanguage>en</NeutralLanguage>
+    <NoWarn>1701;1702;1591;CS7035</NoWarn>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+
+    <AssemblyName>ActiveLogin.Authentication.BankId.AspNetCore.Qr</AssemblyName>
+    <PackageId>ActiveLogin.Authentication.BankId.AspNetCore.Qr</PackageId>
+
+    <VersionPrefix>3.0.0</VersionPrefix>
+    <VersionSuffix>alpha-1</VersionSuffix>
+    <AssemblyVersion>3.0.0</AssemblyVersion>
+    <FileVersion Condition="'$(BUILD_BUILDNUMBER)' != ''">$(VersionPrefix).$(BUILD_BUILDNUMBER)</FileVersion>
+
+    <Description>QR code generation for ActiveLogin.Authentication.BankId.AspNetCore that enables an application to support Swedish BankID's (svenskt BankIDs) native authentication workflow.</Description>
```

Comment:
Use:
"QR code generation for ActiveLogin.Authentication.BankId.AspNetCore using the QRCoder library."